client/e2e/workflows/bceidbusiness/industry_user.spec.ts

```diff
@@ -50,12 +54,16 @@ test.describe("Test Workflow industry_user", () => {
     await dashboardPage.clickSelectOperatorTile();
     // 🔍 Assert current URL is select operator
     await selectOperatorPage.urlIsCorrect();
+    // 🔍 Assert the form is visible - needed to prevent analyzeAccessibility from failing
+    await selectOperatorPage.formIsVisible();
     // 📷 Cheese!
     pageContent = page.locator("html");
     await happoPlaywright.screenshot(page, pageContent, {
       component: "Select operator page",
       variant: "default",
     });
+    // ♿️ Analyze accessibility
+    await analyzeAccessibility(page);
```

Comment:
It looks like every happo shot is accompanied by the accessibility test, so could consider making a helper to do both.

Reply:
Thanks, I have made a separate PR to simplify this and make sure I don't break our Happo screenshots here (still WIP): https://github.com/bcgov/cas-registration/pull/1426